baseline.yaml

```
@@ -129,18 +129,20 @@ criteria:
     maturity_level: 1
     category: Build & Release
     criteria: |
-      The project's build and release pipelines
-      MUST NOT execute arbitrary code that is
-      input from outside of the build script.
+      A project’s build process that either leads to release
+      or can read secret data MUST NOT execute untrusted external code.
```

In the interests of brevity, I'd rewrite this as:

```suggestion
      A build process that either leads to release or can read secret data
      MUST NOT execute untrusted code.
```
The more we can excise unnecessary words from the criteria, the easier it will be for people to quickly read, addressing part of the concern in #99. I can see an argument for leaving "external" in, but I'd counter that we don't want the build pipeline running untrusted external code either, and the objective & implementation fields give us more space to expand on what we're trying to say.

That said, I'm not sure I like explicitly calling out "read secret data" here. Are there other times that untrusted code in the build process is of concern?